README.md

```diff
@@ -2,8 +2,9 @@
 - Install Python version 3.11 (pyenv recommended)
 - Run the `install_dependencies.sh` script to install requirements and setup a virtual environment (`.venv`)
 - Activate the virtual environment `source .venv/bin/activate`
-- Create `.env` file and populate it with API keys:
+- Create `.env` file and populate it with API keys (see "Resources" section for links):
     - `OPENAI_API_KEY`
+    - `SCRAPEOPS_API_KEY`
```

Comment:
Since we figured out how to do this without SCRAPEOPS I think we can delete this and the related resources listed in the resources section.